website_sale_cart_preview/static/src/js/website_sale_cart_preview.js

```diff
@@ -0,0 +1,45 @@
+$(document).ready(function() {
```

Comment:
I couldn't find any documentation whatsoever on this stuff, but I can try. The source seems quite simple.